src/adr/Adr/AdrManager.cs

```diff
@@ -0,0 +1,177 @@
+﻿using adr.Utils;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace adr.Adr
+{
+    internal class AdrManager
```

Comment:
```suggestion
    internal class ArchitectureDecisionLog
```

Reply:
I'm not jealous about the name of my classes, but in this case this one is a central class that provides a lot of different methods for manage ADRs and should probably implements more and more as functionalities will be added...

I therefore believe that we should orient the class name towards something that express this function... what do you think about something like `ArchitectureDecisionProvider`?

Reply:
Reading your comments on #4 I understood better your point of view on how to rename this class (giving it the "dignity" of object, and not only of an utility class), and I like it, so please forget my previous comment

Reply:
Exactly, often when you have to resort to using a "Manager"-class there is a more elegant way of expressing what its role is.

In this case you could feel that their was a need for an overarching structural element to "manage" the flow of the application. You can also see that all the operations the cli exposes operations are against the log of adr's. In the easy (already implemented) cases this involved nothing more than adding one record or creating the directory. But once you started to implement features which touched on multiple records, this naïve setup started to break down.

So it's good that you saw the need, but I think representing the cli functionalities as operations on the `ArchitectureDecisionLog` is going to allow us more flexibility in the future.